code/game/machinery/ecto_sniffer.dm

```diff
@@ -34,12 +40,13 @@
 /obj/machinery/ecto_sniffer/proc/activate(mob/activator)
 	flick("ecto_sniffer_flick", src)
 	playsound(loc, 'sound/machines/ectoscope_beep.ogg', 25)
-	visible_message("<span class='notice'>[src] beeps, detecting ectoplasm! There may be additional positronic brain matrixes available!</span>")
+	var/msg = "[src] beeps, detecting ectoplasm! There may be additional positronic brain matrixes available!"
```

Comment:
This is the most pedantic thing you will ever hear, but matrices would be preferable as it is more consistent with the code-base. 

 Hell my spellchecker gets angry at matrixes, even though it is valid.

Reply:
Fair! Done and did